tools/bazel_tools/shellwrap.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This script is helpful in entering an interactive shell from a bazel build
+# before running a given bazel executable.
+# This can provide a quick way to explore the sandbox directory and filesystem.
+# Typical use is with
+#
+#     bazel run --run_under=//tools/bazel:shell_wrapper //:target
+#     OR
+#     bazel run --config=shell //:target
+
+shell="/bin/bash"
```

Comment:
should shell default to user's shell?

Reply:
we are using a shell specific `--noprofile` and `--rcfile`.
zsh for example doesn't have the `--noprofile` (I think it has but under different name).